itou/www/stats/views.py

```diff
@@ -61,6 +62,21 @@ def public_advanced_stats(request, template_name=_STATS_HTML_TEMPLATE):
     return render(request, template_name, context)
 
 
+@xframe_options_exempt
+def public_iframe_stats(request, dashboard_id, template_name="stats/stats_iframe.html"):
+    """
+    Get iframe stats page
+    Exempt the x-frame options for this view
+    We do it because we need to embed dashboards on pilotage
```

Comment:
`We do it because we want to allow users to download chart data which is only possible via embedded dashboards and not via public dashboards.`